src/java.base/share/native/libjava/System.c

```diff
@@ -297,6 +303,7 @@ static void cpchars(jchar *dst, char *src, int n)
     }
 }
 
+#if 0
```

Comment:
A comment explaining why this is omitted would be welcome, with another on the `#endif` linking it to here.

Reply:
Added a comment.